app/components/concept-page/question-card.hbs

```diff
@@ -4,17 +4,37 @@
       {{markdown-to-html @question.queryMarkdown}}
     </div>
 
-    <div class="flex flex-col -mx-3 sm:-mx-4 gap-px" {{did-insert this.handleDidInsertOptionsList}}>
+    {{! template-lint-disable no-invalid-interactive }}
+    <div
+      class="question-card-option-container flex flex-col -mx-3 sm:-mx-4 gap-px"
+      {{did-insert this.handleDidInsertOptionsList}}
+      {{! @glint-expect-error on-key modifier types aren't right? }}
```

Comment:
We can avoid all of these warnings by using `{{on-key}}` as a helper instead, I don't see a particular reason to attach it to this container element